src/mongo/tree/MongoCollectionTreeItem.ts

```diff
@@ -25,6 +25,7 @@ export class MongoCollectionTreeItem implements IAzureParentTreeItem {
 	private _cursor: Cursor | undefined;
 	private _hasMoreChildren: boolean = true;
 	private _batchSize: number = DefaultBatchSize;
+	private readonly deferToShell = null;
```

Comment:
Make it a const (may need to pull out of class) and add a comment as to what it means